cloud-consumption-api/src/main/java/com/sequenceiq/consumption/api/v1/consumption/model/request/StorageConsumptionRequest.java

```diff
@@ -0,0 +1,34 @@
+package com.sequenceiq.consumption.api.v1.consumption.model.request;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.sequenceiq.consumption.api.doc.ConsumptionModelDescription;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class StorageConsumptionRequest extends ConsumptionBaseRequest {
+
+    @NotNull
```

Comment:
same, maybe `NotEmpty` would fit this better if there can't be more specific validation, like regexp, or size limitation